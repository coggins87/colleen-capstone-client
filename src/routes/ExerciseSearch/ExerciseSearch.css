.ExerciseSearch {
  display: flex;
  flex-direction: column;
  margin: auto;
  
}

.ExerciseHeader {
  margin-top: 30%;
}