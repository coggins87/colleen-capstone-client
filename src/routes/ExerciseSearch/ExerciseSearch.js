import React from 'react'
import SearchForm from '../../components/SearchForm/SearchForm'
import './ExerciseSearch.css'

class ExerciseSearch extends React.Component {

 state = {
   visibleForm: true,
   result: []
 }

 toggleForm = ()=> {
   this.setState({
     visibleForm: !this.state.visibleForm
   })
 }

  render(){
    return (
    <div className ='ExerciseSearch'>
    <h2 className="ExerciseHeader">Generate a Random AMRAP</h2>

    {this.state.visibleForm ? <SearchForm/> : <></>}

    </div>
    )
  }
}

export default ExerciseSearch