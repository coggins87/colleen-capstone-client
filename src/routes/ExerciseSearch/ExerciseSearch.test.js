import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ExerciseSearch from './ExerciseSearch'

describe(`ExerciseSearch component`, () => {


  it('renders the ExerciseSearch component by default', () => {
    const wrapper = shallow(<ExerciseSearch />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})