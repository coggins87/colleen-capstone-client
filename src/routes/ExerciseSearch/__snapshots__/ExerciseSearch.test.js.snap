// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExerciseSearch component renders the ExerciseSearch component by default 1`] = `
<div
  className="ExerciseSearch"
>
  <h2
    className="ExerciseHeader"
  >
    Generate a Random AMRAP
  </h2>
  <SearchForm />
</div>
`;
